akka-docs/rst/scala/cluster-sharding.rst

```diff
@@ -238,41 +239,42 @@ Proxy Only Mode
+method.
 
 Passivation
 -----------
 
-If the state of the entries are persistent you may stop entries that are not used to
+If the state of the entities are persistent you may stop entities that are not used to
 reduce memory consumption. This is done by the application specific implementation of
-the entry actors for example by defining receive timeout (``context.setReceiveTimeout``).
-If a message is already enqueued to the entry when it stops itself the enqueued message
+the entity actors for example by defining receive timeout (``context.setReceiveTimeout``).
+If a message is already enqueued to the entity when it stops itself the enqueued message
 in the mailbox will be dropped. To support graceful passivation without loosing such
-messages the entry actor can send ``ShardRegion.Passivate`` to its parent ``Shard``.
-The specified wrapped message in ``Passivate`` will be sent back to the entry, which is
+messages the entity actor can send ``ShardRegion.Passivate`` to its parent ``Shard``.
+The specified wrapped message in ``Passivate`` will be sent back to the entity, which is
 then supposed to stop itself. Incoming messages will be buffered by the ``Shard``
-between reception of ``Passivate`` and termination of the entry. Such buffered messages
-are thereafter delivered to a new incarnation of the entry.
-
-Remembering Entries
--------------------
-
-The list of entries in each ``Shard`` can be made persistent (durable) by setting
-the ``rememberEntries`` flag to true when calling ``ClusterSharding.start``. When configured
-to remember entries, whenever a ``Shard`` is rebalanced onto another node or recovers after a
-crash it will recreate all the entries which were previously running in that ``Shard``. To
-permanently stop entries, a ``Passivate`` message must be sent to the parent the ``Shard``, otherwise the
-entry will be automatically restarted after the entry restart backoff specified in the configuration.
-
-When ``rememberEntries`` is set to false, a ``Shard`` will not automatically restart any entries
-after a rebalance or recovering from a crash. Entries will only be started once the first message
-for that entry has been received in the ``Shard``. Entries will not be restarted if they stop without
+between reception of ``Passivate`` and termination of the entity. Such buffered messages
+are thereafter delivered to a new incarnation of the entity.
+
+Remembering Entities
+--------------------
+
+The list of entities in each ``Shard`` can be made persistent (durable) by setting
+the ``rememberEntities`` flag to true in ``ClusterShardingSettings`` when calling 
+``ClusterSharding.start``. When configured to remember entities, whenever a ``Shard`` 
+is rebalanced onto another node or recovers after a crash it will recreate all the
+entities which were previously running in that ``Shard``. To permanently stop entities, 
+a ``Passivate`` message must be sent to the parent the ``Shard``, otherwise the
```

Comment:
ah, `Passivate` should be sent to the parent actor, period. That actor is the `Shard` actor, but that not really something that the user should care about.